### PR TITLE
fix: Top Collection representation "streched" (only Base)

### LINKED ISSUE #10412: Top Collection representation "streched" (only Base)
If you go to "Top Collection" it works for PolkadotHub, KusamaHub and Kusama but not for base.

Working example (Polkadot):
grafik

Base:
grafik

The button to choose time period also disappears:
grafik

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

>The button to choose time period also disappears:

atm  volume on base is not tracked, that's why it was decided to hide the period buttons 

- [x] Closes #10412

## Needs QA check

- @kodadot/qa-guild please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-06-06 at 13 08 49@2x

### CODE REVIEW

**Comment by vikiival** [comment]
cc @Jarsen136 @preschian

**Comment by vikiival** [comment]
TL;DR of your changes ?

**Comment by hassnian** [comment]
> TL;DR of your changes ?

there was a bug in the `fillRows` feature, if `fillRows` can't be fulfilled now it will fallback to the normal generated `cols` amount

**Review by vikiival — APPROVED** [review]
